our-work/collaborations/dennery.yml

```diff
@@ -1,8 +1,8 @@
 title: Dennery Lab
 id: dennery-lab
 description: The Dennery lab investigates neonatal lung injury and repair. CCV's collaboration with the Dennery lab involved analyzing and publishing ChIP-Seq and single cell RNA-seq data.
-tags:
-  - active
+active: True
+department:
   - computational-biology
```

Comment:
@ashokrags we are separating tags from department. Would you advise on what to have here on each o those categories?